languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -46521,6 +46521,42 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
     <!-- Grammatik -->
     <!-- ====================================================================== -->
     <category id="GRAMMAR" name="Grammatik">
+        <rulegroup id="HANDELN_DARUM_DARUEBER" name="handeln darum/darüber (davon)" default="temp_off">
```

Comment:
Ich überlege gerade, ob der Gebrauch der standardsprachlichen Präposition wirklich ein Grammatikfehler ist. Vielleicht doch eher eine Registersache?

Reply:
@udomai 
Ja, evtl.  
Die Frage wäre aber in welchen Situationen, Dialekten etc. ist eine andere Präposition korrekt, sodass es kein 'harter' Grammatikfehler ist?

Reply:
Rein linguistisch ist die Frage müßig, sobald feststeht, **dass** es so gesagt wird.
Zudem handelt es sich um kein morphologisches Problem, und auch nicht um einen Fehler, der das Verständnis beeinträchtigt. Das sind alles Faktoren, die diese Regel in meinen Augen eher zu einer Stil- oder Registerregel machen.

Reply:
Ok, können das gerne zu Stil packen.

Reply:
Gut, machst du das?

Reply:
Alles klar!

Reply:
Könnte gemergt werden.

Reply:
@udomai ja, bei mir ist sie in STYLE